### DISCUSSION TITLE
pptx format ignoring comparison layout

### DISCUSSION THREAD

**Discussion by yonisidi** [quarto-dev/quarto-cli#10050] [Q&A] [answered]
I have a slide that looks like this

````
## My Header

::: columns
::: {.column}
- @fig-myfig shows something amazing. 
- bullet 2
- bullet 3
:::
::: {.column}
```{r fig-myfig}
#| fig-cap: 'my caption'
knitr::include_graphics('my-figure.png')
```
:::
:::
````

when i render the slides the layout that is chosen by quarto/pandoc is `Two Content` instead of `Comparison`. According to the docs it should be the latter.

Do I need to do something else in the script to get it to use the `Comparison` layout?

Documentation:
> Two Content - This layout is used for two-column slides, i.e. slides containing a div with class columns which contains at least two divs with class column.
> Comparison - This layout is used instead of “Two Content” for any two-column slides in which at least one column contains text followed by non-text (e.g. an image or a table).

<details><summary>_quarto.yml</summary>

```yml
project:
  type: default
format: pptx
```
</details>

**Comment by mcanouil** [comment, marked as answer]
Your example does not follow the documentation for the "comparison" layout.
You have a text column and an image column.

The column needs to have "text followed by non-text".

````qmd
---
format: pptx
---

## My Header

::: columns
::: {.column}
- bullet 2
- bullet 3
:::

::: {.column}
- @fig-myfig shows something amazing. 

![An image](https://placehold.co/600x400.png){#fig-myfig}
:::
:::
````

**Reply by yonisidi** [reply]
Ah ok.

That makes sense. 

Thank you for the clarification, I didn't understand from the documentation that the text has to be in the same column as the figure.
